moto/rds/models.py

```diff
@@ -1132,6 +1154,25 @@ def __init__(
         self.multi_az = multi_az
         self.db_subnet_group_name = db_subnet_group_name
         self.db_security_groups = db_security_groups or []
+        self.vpc_security_group_ids = vpc_security_group_ids or []
```

Comment:
Looks like a failed master merge after #9128 - shouldn't be any modification to this attribute in your PR.